CHANGELOG.adoc

```diff
@@ -2,6 +2,12 @@
 
 == DFX
 
+=== feat!: Remove the wallet proxy and the --no-wallet flag
+
+Breaking change: Canister commands, except for `dfx canister create`, will make the call directly, rather than via the user's wallet. The `--no-wallet` flag is thus removed from `dfx canister` as its behavior is the default.
+
+When upgrading existing canisters, use the `--wallet` flag in conjunction with `dfx identity get-wallet` in order to restore the old behavior.
+
```

Comment:
We need to spell out in detail what commands developers need to execute in order to update the controllers of their existing canisters.